.github/workflows/tests.yaml

```diff
@@ -41,42 +52,23 @@ jobs:
           cache: true
       - name: Build devbox
         run: go install ./cmd/devbox
-      - name: Install additional shells
+      - name: Install additional shells (dash, zsh)
         run: |
-          sudo apt-get update
-          sudo apt-get install dash zsh
+          if [ "$RUNNER_OS" == "Linux" ]; then
```

Comment:
I went with the env var because I thought it looked weird always seeing a skipped step. If this grows to include more OS-specific commands, then I agree it would be worth splitting up.